watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/peracetic_acid_disinfection/tests/test_peracetic_acid_disinfection_sweep.py

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import os
+import pytest
+from watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.peracetic_acid_disinfection.peracetic_acid_disinfection_sweep import (
+    run_analysis,
+)
+
+sweep_list = []
+for case_num in [1]:
+    sweep_list.append(case_num)
+
+
+@pytest.mark.parametrize("case_num", sweep_list)
+@pytest.mark.integration
+def test_sweep(case_num, tmp_path):
```

Comment:
Can you confirm that `tmp_path` here doesn't cause issues with test?

Reply:
It won't -- this is an included pytest fixture for a temporary path (i.e., the files will be removed after the test completes).